lib/Doctrine/ORM/Query/SqlWalker.php

```diff
@@ -162,9 +162,11 @@ class SqlWalker implements TreeWalker
     private $platform;
 
     /**
-     * {@inheritDoc}
+     * @param AbstractQuery $query
```

Comment:
Can be dropped - our coding standard enforces removal of all redundant docblock parameters